src/src/ErrorDisplay.js

```diff
@@ -4,14 +4,14 @@ import Toast from 'components/Toast/index.js'
 
 const ErrorDisplay = () => {
   const { error } = useContext(ErrorContext);
-  
+
   if (error.length === 0 ) {
     return null;
   }
 
   return (
     <div>
-        {(error.map(e => <Toast message={e}></Toast>))}   
+        {(error.map((e, index) => <Toast key={index} message={e}></Toast>))}
```

Comment:
What does the `key={index}` thing do :)

Reply:
https://react.dev/learn/rendering-lists#why-does-react-need-keys

TLDR: when rendering an array of components, keys allow React to differentiate between the components

Unfortunately the e variable in this case in the actual error message, and there can occasionally be duplicates, so I instead opted for using the index value that tells us the location of e within the array that is generated by .map

Reply:
ok cool, thanks for the link